yarn.lock

```diff
@@ -8195,6 +8195,13 @@ testdouble@^3.2.6:
     quibble "^0.5.1"
     stringify-object-es5 "^2.5.0"
 
+testem-failure-only-reporter@^0.0.1:
+  version "0.0.1"
+  resolved "https://registry.yarnpkg.com/testem-failure-only-reporter/-/testem-failure-only-reporter-0.0.1.tgz#d096a22b3c78b4d4bdc74d4f8727d75ef8c57b7a"
+  integrity sha512-a3bquFhuguXcI244mAzt8ByCIEoAOsI7NVihNafi8PTYDVCnpgxm73JluduR9GITXopdOeyLAJnlI3pavY7x3w==
+  dependencies:
+    testem "^2.9.2"
```

Comment:
I'm a little worried about this being a proper dependency. I think in this case it might make sense to have it as a peer dependency? 🤔